pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>de.thbingen.info2</groupId>
+    <artifactId>HalloTrain</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
```

Comment:
Maven beschwert sich:

> [WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!

Das geht weg, wenn man in den `properties` hier `project.build.sourceEncoding` auf `UTF-8` setzt.

Reply:
ich hoffe ich habe die Änderungswünsche richtig verstanden.

Reply:
Kannst du ja selber prüfen - kommt die Warnung noch?

Reply:
mir springt keine mehr ins Auge.

Reply:
Dann hat's wohl funktioniert. :wink: (Ich sehe auch keine mehr.)